basil/utils/sim/SiLibUsbSbusDriver.py

```diff
@@ -0,0 +1,177 @@
+#
```

Comment:
I think this file is not needed ?

Reply:
For completeness?
Don't know.

Reply:
This driver is for a physical bus that actually MIO2 board has (actual pins of FPGA). It will never change.

Reply:
I see, ok.